pyClient/README.md

```diff
@@ -45,11 +55,192 @@ and fails to fetch it (or fails to fetch the right version).
+$ zeth notes
+```
+lists information about all known notes belonging to the current user.
+
+## Spending and withdrawing notes
+
+`zeth send` and `zeth withdraw` require inputs in the form of Zeth note
+commitments, and outputs (for `zeth send`) in the form of Zeth addresses and
+amounts.
+
+**Inputs** are Zeth notes owned by the current client, which should be visible
+via `zeth notes`.  Either the integer "address" or the truncated commitment
+value (8 hex chars) can be used to specify which notes to use as inputs.
+
+**Outputs** are given as the Zeth public key and a value, separated by a comma
+`,`.  The form of the public key is exactly as in the `zeth-key.json.pub` file.
+That is, two 32 byte hex values separated by a colon `:`.
+
+Examples:
+```console
+$ zeth notes
+b1a2feaf: value=200, addr=0
+eafe5f84: value=100, addr=2
+
+$ zeth spend \
+    --in eafe5f84 \                       # "eafe5f84: value=100, addr=2"
+    --in 0 \                              # "b1a2feaf: value=200, addr=0"
+    --out d77f...0e00:cc7c....7f76,120 \  # 120 to this addr
+    --out 3a43...fd3b:9fc8....b838,180    # 180 to this addr
+
+$ zeth withdraw --in eafe5f84
+```
+
+`zeth spend` will fail if the total value of inputs does not match the value of
+outputs. To return any change to the client, specify an output using the value
+of `zeth-key.json.pub` and the correct change amount.
+
+## Async transactions
+
+The `deposit`, `spend` and `withdraw` operations broadcast transactions to the
+Ethereum network.  By default, these operations output the transaction ID that
+was broadcast.  This Id can be passed to `zeth sync` via the `--wait-tx` flag,
+forcing this command to wait for that transaction to be committed before
+searching for new notes.  Alternatively, the `--wait` flag can be passed to the
+`deposit`, `spend` and `withdraw` operations to make them wait and sync new
+notes before exiting.
+
+## Limitations - Note and Key management
+
+As proof-of-concept softare, these tools are not suitable for use in a
+production environment and have several functional limitations.  Some of those
+limitations are mentioned here.
+
+The `zeth` tool suite does not track which of the client's notes have been spent
+by previous operations.  In the presence of async transactions and possible
+forks in the chain, such tracking logic would greatly increase the complexity of
+the client tools and is considered out of scope for this proof-of-concept.  The
+user must manually track which notes have been spent (for example by moving
+their files into a `spent` subdirectory where they will not be seen by the
+wallet).
+
+All values that make up the zeth secret key, and secrets (required to spend
+notes) are stored in plaintext.  A fully secure client would encrypt these to
+protect them from malicious entities that may gain access to the file system.
+Such client-side security mechanisms are also beyond the scope of this
+proof-of-concept implementation.
```

Comment:
You can also add that backups for the keys would be necessary but outside of the scope as this project is still in PoC stage.

Reply:
Added a note about this.